episodes/introduction.md

```diff
@@ -30,17 +30,59 @@ Objectives
 
 ## Introduction
 
-Placeholder text. 
+Placeholder text.
+
+:::{image} ../media/episodes/introduction/hub-overview.png
+:width: 100%
+:alt: Diagram showing an overview of a 2i2c Hub, includiing authentication, online content, custom environments, the community hub, cloud infrastructure, support and services.
+
+:::
 
 ## Exercises
 
-::::{dropdown} Challenge Title
+::::{dropdown} Which of the following statements are True/False?
 :open:
 
-Challenge content
+a. The infrastructure for running a 2i2c-managed hub is closed source and not replicable across different cloud providers.  
+b. Access to the hub can be controlled with authentication and authorisation.  
+c. Software environments are inflexible and irreproducible.  
+d. 2i2c provides operations support for your hub.
```

Comment:
```suggestion
a. 2i2c-managed hubs are only available via a single cloud provider.  
b. Access to the hub can be controlled with authentication and authorisation.  
c. Software environments are inflexible and irreproducible.  
d. 2i2c provides operations support for your hub.
```

Reply:
```suggestion
a. The infrastructure for running a 2i2c-managed hub is closed source and not replicable across different cloud providers.  
b. A hub can be accessed and used by anyone who knows its URL.  
c. Software environments are inflexible and irreproducible.  
d. 2i2c provides operations support for your hub.
```

Reply:
A couple of suggestions to make these answer feel slightly more difficult to guess. Feel free to discard, and note that my changes flip the truth of both answers.

Reply:
Not sure what will happen if you apply both of the above suggestions: maybe I should have created a single suggestion containing both changes...?

Reply:
I like both of your suggestions! The two suggestions will override each other, so I'll incorporate both changes into a single suggestion below and add that to a commit.

Reply:
```suggestion
a. 2i2c-managed hubs are only available via a single cloud provider.  
b. A hub can be accessed and used by anyone who knows its URL.  
c. Software environments are inflexible and irreproducible.  
d. 2i2c provides operations support for your hub.
```